src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeRepositoryTest.java

```diff
@@ -183,6 +189,32 @@ void should_return_empty_optional_when_envelope_for_given_container_and_file_nam
         assertThat(result).isEmpty();
     }
 
+    @Test
+    void should_find_within_date_range() {
+        // given
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        Instant instant1 = LocalDateTime.parse("2019-12-19 10:31:25", formatter).toInstant(ZoneOffset.UTC);
+        Instant instant2 = LocalDateTime.parse("2019-12-20 11:32:26", formatter).toInstant(ZoneOffset.UTC);
+        Instant instant3 = LocalDateTime.parse("2019-12-20 12:33:27", formatter).toInstant(ZoneOffset.UTC);
+        Instant instant4 = LocalDateTime.parse("2019-12-21 13:34:28", formatter).toInstant(ZoneOffset.UTC);
+        repo.insert(new NewEnvelope("a", "b", instant1, instant1, Status.DISPATCHED));
+        repo.insert(new NewEnvelope("c", "d", instant2, instant2, Status.DISPATCHED));
+        repo.insert(new NewEnvelope("e", "f", instant3, instant3, Status.DISPATCHED));
+        repo.insert(new NewEnvelope("g", "h", instant4, instant4, Status.DISPATCHED));
```

Comment:
you've set the same instant in two fields of envelope, test name or test data don't explain which property of envelope is used for search